tests/functional/recovery_key.js

```diff
@@ -197,14 +199,60 @@ registerSuite('Recovery key', {
   }
 });
 
+registerSuite('Recovery key - experiment', {
+  afterEach: function () {
+    return this.remote.then(clearBrowserState());
+  },
+
+  tests: {
+    'verified - control': function () {
+      email = TestHelpers.createEmail('sync{id}');
+      const queryOptions = {query: {forceExperiment: 'recoveryKey', forceExperimentGroup: 'control'}};
+
+      return this.remote
+        .then(openPage(SIGNUP_URL, selectors.SIGNUP.HEADER, queryOptions))
+        .then(fillOutSignUp(email, PASSWORD))
+        .then(testElementExists(selectors.CONFIRM_SIGNUP.HEADER))
+        .then(openVerificationLinkInSameTab(email, 0))
+        .then(testElementExists(selectors.SETTINGS.HEADER))
+
+        // Perform a slight sleep here to ensure that the checks
+        // to enable the panel have been performed.
+        .sleep(1000)
```

Comment:
replace it with a JS selector to check the height of the panel or something like that

Reply:
> replace it with a JS selector to check the height of the panel or something like that

Even better

Reply:
Tried using the selector but got `stale element exception`. However, using the similar pattern as obscure element, I was able to get this passing 5/5 times. It was also faster because it would only sleep when it needed too.